packages/DataGrid/src/DataGrid.js

```diff
@@ -0,0 +1,628 @@
+            outerElementType={outerElementType}
+            innerElementType={innerElementType}
+            className={`${gridId}-header`}
+          >
+            {({ columnIndex, style }) => {
+              const { header, headerProps } = ColumnDefinitions[columnIndex].props;
+
+              if (stickyColumnsIndexes.includes(columnIndex)) return null;
+              const { style: styleProps = {}, ...moreProps } =
+                typeof headerProps === "function" ? headerProps({ header }) : {};
+
+              return (
+                <styled.CellHeader role="columnheader" style={{ ...style, ...styleProps }} {...moreProps}>
+                  {typeof header === "function" ? header() : header}
+                </styled.CellHeader>
+              );
+            }}
+          </Grid>
+        </styled.Flex>
+        {/** STICKY COLUMNS */}
+        <styled.Flex>
+          <Grid
+            className={`${gridId}-sticky-columns`}
+            columnCount={stickyColumnsIndexes.length}
+            columnWidth={columnIndex => {
+              return ColumnDefinitions[stickyColumnsIndexes[columnIndex]].props.width;
+            }}
+            height={calculatedTableHeight - scrollBarWidth}
+            innerElementType={innerElementType}
+            onItemsRendered={handleItemsRedered}
+            onScroll={handleScrollStickyColumns}
+            outerElementType={outerElementType}
+            overscanColumnCount={overscanColumnCount}
+            overscanRowCount={overscanRowCount}
+            rowCount={rowCount}
+            rowHeight={() => rowHeight}
+            width={stickyGridWidth}
+          >
+            {({ columnIndex, rowIndex, style }) => {
+              const column = ColumnDefinitions[stickyColumnsIndexes[columnIndex]].props;
+              const cellA11yText =
+                typeof column.cell === "function"
+                  ? column.cellA11yText && column.cellA11yText({ row: data[rowIndex], rowIndex, columnIndex })
+                  : data[rowIndex][column.cell];
+              const headerA11yText = columnHeadersA11yText[columnIndex];
+              const a11yText = a11yTextMessage(cellA11yText, headerA11yText, rowIndex);
+
+              return (
+                <Cell
+                  a11yText={a11yText}
+                  column={column}
+                  columnIndex={columnIndex}
+                  data={data}
+                  gridId={gridId}
+                  ref={handleRefCell({ columnIndex, rowIndex })}
+                  rowIndex={rowIndex}
+                  style={style}
+                />
+              );
+            }}
+          </Grid>
+
+          {/** COLUMS */}
+
+          <Grid
+            className={`grid-${gridId}`}
+            columnCount={columnCount}
+            columnWidth={columnIndex => {
+              if (stickyColumnsIndexes.includes(columnIndex)) return 0;
+              return ColumnDefinitions[columnIndex].props.width;
```

Comment:
Is one line nicer?

    return stickyColumnsIndexes.includes(columnIndex) ? 0 : ColumnDefinitions[columnIndex].props.width;